### ISSUE TITLE
The association App\Entity\Product\Product#productType refers to the inverse side field BitBag\SyliusMolliePlugin\Entity\ProductType#productType which does not exist.

### ISSUE THREAD

**Issue by DieterHolvoet** [BitBagCommerce/SyliusMolliePlugin#68]
When looking through the Symfony Profiler for an unrelated issue, I noticed an error in the Entities Mapping section of the Doctrine pane:
```
The association App\Entity\Product\Product#productType refers to the inverse side field BitBag\SyliusMolliePlugin\Entity\ProductType#productType which does not exist.
```

**Comment by daniel8702** [comment]
Hi @DieterHolvoet, thanks for Your message. 

We fix that on the next release :)

**Comment by daniel8702** [comment]
fixed on 3.3.5

**Comment by Vandaahl** [comment]
I am running into this issue on 3.3.5. Installed the plugin by carefully following the installation instructions (using annotations).
